apps/namadillo/src/atoms/staking/services.ts

```diff
@@ -129,6 +129,9 @@ export const createClaimAndStakeTx = async (
     props: ClaimRewardsProps | BondProps
   ): Promise<TxMsgValue> => {
     if ("amount" in props) {
+      // We have to force it in case: current balance < rewards to claim
+      // This will still log the error msg in the terminal, unfortunately we can't do much about it
+      wrapperTxProps.force = true;
```

Comment:
Interesting. Will this potentially break deserialization in the extension? If it simply ignores it, I think we're good!